ssg/build_remediations.py

```diff
@@ -306,7 +306,7 @@ def parse_from_file_with_jinja(self, env_yaml):
                 all_conditions += " && ".join(inherited_conditionals)
             if rule_specific_conditionals:
                 if all_conditions:
-                    all_conditions += " && ( " + " || ".join(rule_specific_conditionals) + " )"
+                    all_conditions += " && { " + " || ".join(rule_specific_conditionals) + "; }"
```

Comment:
I thought this would cause syntax issues if a rule has inherited platforms and its specific platform is `machine`.
But it turned out I had introduced the syntax error, :laughing: .

Nice work, :)